src/__tests__/cacao.test.ts

```diff
@@ -1,17 +1,32 @@
 import { Wallet } from '@ethersproject/wallet'
-import { Cacao, CacaoBlock } from '../cacao'
-import { SiweMessage } from '../siwe'
+import { Cacao, CacaoBlock } from '../cacao.js'
+import * as solanaWeb3 from '@solana/web3.js'
+import { SiweMessage } from '../siwx/siwe.js'
+import { SiwsMessage } from '../siwx/siws.js'
+import nacl from 'tweetnacl'
+import base58 from 'bs58'
 
 describe('Cacao', () => {
-  const wallet = Wallet.fromMnemonic(
+  const ethWallet = Wallet.fromMnemonic(
     'despair voyage estate pizza main slice acquire mesh polar short desk lyrics'
   )
-  const address = wallet.address
+  const ethAddress = ethWallet.address
 
-  test('Can create and verify Cacao Block', async () => {
+  const solanaSecretKey = new Uint8Array([
+    146, 224, 142,  57, 174, 232, 125,  83, 254,  38,  57,
```

Comment:
Just encode this as hex and decode to Uint8Array using the `uint8arrays` pkg to avoid the lint error.